pages/Network_APIs/Live_Media_Distribution/Requirements_for_Live_Media_Distribution.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Requirements
+parent: Live Media Distribution
+grand_parent: Network APIs
+nav_order: 1
+has_children: false
+---
+
+{: .warning }
+This documentation is currently **under development and subject to change**. It reflects outcomes elaborated by 5G-MAG members as part of **WI: Network Exposure and APIs Supporting Media Services and Applications**
+We welcome and encourage contributions from the broader community. If you are interested in becoming a member of the 5G-MAG and actively participating in shaping this work, please contact the [Project Office](https://www.5g-mag.com/contact)
+
+# Network Capabilities (Network Services) for Live Segmented Media Distribution
+
+## Quality of Service
+The following requirements are defined:
+- Ability to request different QoS profiles for individual data flows being distributed across a target service area
+- Ability to provision individual data flows for realiably low latency (distribution delay)
+- Delivery to endpoint (Application Media Server) may be identified by security/protocol/IP/port
+- Ability to configure new or re-configure existing QoS profiles to be selected during runtime 
```

Comment:
This one needs a bit of work.
- What does "during runtime" mean? Seems vague. Does it mean during a live media consumption session? I can see that might be applicable in the case of dynamic adaptation by the client, which is common for live video distribution but less common for live audio distribution.
- Which party is responsible for the (re)selection of the QoS profiles? Is this intended to be application-driven or network-driven?